anonym-ip_test.go

```diff
@@ -0,0 +1,341 @@
+package anonymization
+
+import (
+	"bytes"
+	"net"
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	// initialize whatever needs to be initialized
+	os.Exit(m.Run())
+}
+
+func TestEncryptIP(t *testing.T) {
+	// set-up if needed
+	t.Run("IPv4, passphrase", func(t *testing.T) {
```

Comment:
why not having separate function (like TestEncryptIPv4Passphrase)? Any advantage?

Reply:
grouping tests in tests suites. see:

https://golang.org/pkg/testing/
"Subtests and Sub-benchmarks"

you can run only sub-tests of a group of tests:
`go test -run EncryptIP/IPv4`